ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -19,8 +19,9 @@ open Threadext
 module D = Debug.Make(struct let name = "xapi_pvs_proxy_control" end)
 open D
 
-let proxy_port_name bridge =
-  "pvs-" ^ bridge
+let proxy_port_name vif =
+  let uuid = String.sub vif.API.vIF_uuid 0 8 in
```

Comment:
Okay. Well, a clash is still pretty unlikely, and it's not a big problem if there is a clash.